crates/oxc_language_server/src/linter/server_linter_js_plugins_test.rs

```diff
@@ -0,0 +1,38 @@
+#[cfg(test)]
+mod test {
+    use std::fs;
+    use std::path::PathBuf;
+    use std::time::{SystemTime, UNIX_EPOCH};
+
+    use serde_json::json;
+    use tower_lsp_server::{UriExt, lsp_types::Uri};
+
+    use super::super::server_linter::ServerLinterBuilder;
+
+    #[test]
+    fn test_new_with_js_plugins_in_config_does_not_crash() {
```

Comment:
This is not really testing anything, only if the `ServerLinterBuilder::build` did not panic. 
`jsPlugins` for the language server is not implemented, but a milestone goal:  https://github.com/oxc-project/oxc/issues/14826.

Can you remove this file?